packages/coreutils/src/settingregistry.ts

```diff
@@ -20,8 +20,8 @@ import {
 } from '@phosphor/signaling';
 
 import {
-  IDataConnector, StateDB
-} from '.';
```

Comment:
I don't like importing from `'.'`, since it hides where things are actually coming from.